### ISSUE TITLE
Several issues after installing new Qfield Sync pluginn v4.8.0 

### ISSUE THREAD

**Issue by bakkerss** [opengisch/QField#5083]
Apologies for creating a second issue for the same problem but I encountered a second error which also seems to be related to the sync plugin. I couldn't find the first issue I just created so I couldn't edit this one.
 

**First error:** 

After installing the latest release of the plugin I get a python error when I try to package my project (After pressing the create button).

AssertionError 
Traceback (most recent call last):
  File "C:\Users/SebastiaanBakker/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\qfieldsync\gui\package_dialog.py", line 161, in package_project
    offline_convertor = OfflineConverter(
  File "C:\Users\SebastiaanBakker\AppData\Roaming\QGIS\QGIS3\profiles\default\python\plugins\qfieldsync\libqfieldsync.whl\libqfieldsync\offline_converter.py", line 114, in __init__
AssertionError

Python versie: 3.9.5 (tags/v3.9.5:0a7dcbd, May  3 2021, 17:27:52) [MSC v.1928 64 bit (AMD64)] 
QGIS versie: 3.28.4-Firenze Firenze, fd0fb72ab3 

Python pad:
C:/OSGeo4W/apps/qgis-ltr/./python
C:/Users/SebastiaanBakker/AppData/Roaming/QGIS/QGIS3\profiles\default/python
C:/Users/SebastiaanBakker/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins
C:/OSGeo4W/apps/qgis-ltr/./python/plugins
C:\OSGeo4W\apps\grass\grass82\etc\python
C:\Users\SebastiaanBakker\OneDrive - Natuurlink\Documenten
C:\OSGeo4W\bin\python39.zip
C:\OSGeo4W\apps\Python39\DLLs
C:\OSGeo4W\apps\Python39\lib
C:\OSGeo4W\bin
C:\OSGeo4W\apps\Python39
C:\OSGeo4W\apps\Python39\lib\site-packages
C:\OSGeo4W\apps\Python39\lib\site-packages\win32
C:\OSGeo4W\apps\Python39\lib\site-packages\win32\lib
C:\OSGeo4W\apps\Python39\lib\site-packages\Pythonwin
C:/Users/SebastiaanBakker/AppData/Roaming/QGIS/QGIS3\profiles\default/python
C:\Users\SebastiaanBakker\AppData\Roaming\QGIS\QGIS3\profiles\default\python\plugins\qfieldsync\libqfieldsync.whl
C:\Users/SebastiaanBakker/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\qgis_resource_sharing
C:\Users\SebastiaanBakker\AppData\Roaming\QGIS\QGIS3\profiles\default\python\plugins\qgis_resource_sharing
C:\Users\SebastiaanBakker\AppData\Roaming\QGIS\QGIS3\profiles\default\python\plugins\qgis_resource_sharing\ext_libs
C:\Users\SebastiaanBakker\AppData\Roaming\QGIS\QGIS3\profiles\default\python\plugins\qfieldsync\libqfieldsync.whl
C:/Users/SebastiaanBakker/Natuurlink B.V/Natuurlink - Documenten/QGIS/Woonstichting Lieven de Key

**Second error**

This error appeared in the background and I do not recall what I was doing exactly so I can't really reproduce the error

2024-03-08T10:23:15     WARNING    Traceback (most recent call last):
              File "C:\Users/SebastiaanBakker/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\qfieldsync\qfield_sync.py", line 63, in createWidget
              return ProjectConfigurationWidget(parent)
              File "C:\Users/SebastiaanBakker/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\qfieldsync\gui\project_configuration_widget.py", line 113, in __init__
              self.reloadProject()
              File "C:\Users/SebastiaanBakker/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\qfieldsync\gui\project_configuration_widget.py", line 196, in reloadProject
              self.forceAutoPush.setChecked(self.__project_configuration.force_auto_push)
             AttributeError: 'ProjectConfiguration' object has no attribute 'force_auto_push'

**Comment by bakkerss** [comment]
Regarding the second error: I just got it again when I opened a layer's properties. An error window appears for just a very brief moment after which the properties window appears. This window is mixed up with a second window showing the Qfield sync properties (See attachement).

![Qfield issue 5083 - Print screen of properties window ](https://github.com/opengisch/QField/assets/45461027/ad5b40b1-7f01-42da-820f-79914b5377c9)
